tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "lib": ["dom", "es6"],
     "importHelpers": true,
     "paths": {
+      "@angular/*": ["../node_modules/@angular/*"],
```

Comment:
add comment here that mention the issue number + it should be removed when we use angular 5

Reply:
``` 
   "paths": {
      // fix for #605
      // TODO: Remove it after upgrading Angular to v.5
      "@angular/*": ["../node_modules/@angular/*"],

      "@ngx-formly/core": ["./core/index"],
      "@ngx-formly/bootstrap": ["./ui-bootstrap/index"],
      "@ngx-formly/material": ["./ui-material/index"]
    },
```

Something like this?

Reply:
yep